Code/Tools/ProjectManager/Platform/Windows/ProjectBuilderWorker_windows.cpp

```diff
@@ -125,7 +125,7 @@ namespace O3DE::ProjectManager
         m_buildProjectProcess->start(
             "cmake",
             QStringList{ "--build", QDir(m_projectInfo.m_path).filePath(ProjectBuildPathPostfix), "--target",
-                         m_projectInfo.m_projectName + ".GameLauncher", "Editor", "--config", "profile" });
+                         m_projectInfo.m_projectName + ".GameLauncher", "Editor", "--config", "profile", "-m" });
```

Comment:
This is missing the "--" between the cmake and msbuild args here however I don't think we want to blindly force an all core build.  There have been reports of running out of heap space doing this on some machines.

Reply:
I think it will be a constant fight on flags we put here...
If we put something like `/m:2` it will make it slower in machines that have enough memory. If we put `/m` it may run out of heap space on machines that dont have enough memory.
Can this be a setting from the ProjectManager? ideally, we can detect the situation, but a setting can allow to tweak it.
(No need to do it as part of this change)

Reply:
While you are here, can you add `-DLY_UNITY_BUILD=1` to line 79?

Reply:
I really like the idea of being able to configure the build commands in Project Manager or in some data-oriented way, that's definitely on the wishlist

Reply:
if we add presets, then this can pickup those too

Reply:
Sounds like an awesome idea, perfect to be added to the suggest features list, removing the flag for now though.